pkg/webui/console/components/payload-formatters-form/index.js

```diff
@@ -261,6 +336,7 @@ class PayloadFormattersForm extends React.Component {
         initialType === TYPES.GRPC ? initialParameter : getDefaultGrpcServiceFormatter(uplink),
     }
     const options = allowReset ? formatterOptionsWithReset : formatterOptions
+    const defaultFormatter = defaultType.replace('FORMATTER_', '').toLowerCase()
```

Comment:
```suggestion
    const defaultFormatter = defaultType.replace('FORMATTER_', '').toLowerCase()
```

Not a fan of solutions like these. It abuses our internal naming, which is completely arbitrary, as user-facing communication. Someone working on this at a later stage might not be aware of this liability and choose namings that are not OK for being displayed to users. Additionally, these strings are not internationalized and only work in English.

A better solution would be to create a mapping of formatter types to i18n messages and use that instead, similar to what we are doing with `console/lib/field-description-messages.js`.